packages/styles/settings-tools/_s.notification.scss

```diff
@@ -0,0 +1,37 @@
+$theme-background-colors: (
+  'information': $color-secondary-blue-100,
+  'success': $color-primary-01-100,
+  'warning': $color-secondary-orange-100,
+  'danger': $color-secondary-red-100,
```

Comment:
The best way to add colors is used token files in json for colors. You have to create one.